internal/routing/router.go

```diff
@@ -54,5 +55,7 @@ func userRouter(databaseMgr *managers.DatabaseMgr, jwtMgr *managers.JWTMgr, mail
 		r.Post("/login", userHdl.LoginUser)
 		r.Post("/{username}/activate", userHdl.ActivateUser)
 		r.Delete("/{username}/activate", userHdl.ResendToken)
+		r.Post("/{username}/nickname", userHdl.ChangeNickname)
+		r.Post("/{username}/password", userHdl.ChangePassword)
```

Comment:
Wie im vorigen Kommentar, das bitte ändern zu:

```go
r.Put("/users", userHdl.ChangeNicknameAndStatus)
r.Patch("/users", userHdl.ChangePassword)
```

Die entsprechende Dokumentation siehst du in [Postman](https://app.getpostman.com/join-team?invite_code=47970e6341f3956a182e9a0aea00a2e6&target_code=2e812c98237d3e1ecd1a1fb88a66edb2).

Als Erklärung:

- Wir differenzieren nicht mit `nickname` und `password` die Endpunkte, sondern anhand der Methoden. Da `POST /users` schon mit der Registrierung belegt ist und auch iwie nicht zu REST passt, benutzen wir `PATCH /users` für password-only und `PUT /users` für alle anderen Informationen. Alle anderen Informationen bezieht sich momentan nur auf Nickname und Status.
- `{username}` als Path-Parameter brauchen wir nicht, da das ja ein autorisierter Endpunkt ist und wir die Information über den `JWT` kriegen.